paasta_tools/utils.py

```diff
@@ -1889,12 +1854,6 @@ class ResourcePoolSettings(TypedDict):
 PoolToResourcePoolSettingsDict = Dict[str, ResourcePoolSettings]
```

Comment:
this and the autoscaler stuff above also sound like old paasta autoscaler code that can be deleted

Reply:
there's also some stuff below with Slave and whatnot in the name that is mesos-only